src/lib/components/events/event-item.svelte

```diff
@@ -57,16 +64,18 @@
         href={info.meetingLink}
         role="button"
         target="_blank"
-        rel="noopener noreferrer">Join</a
+        rel="noopener noreferrer"
+        on:click={(/* janky hack */) => {
```

Comment:
Without this _hack_ if the user clicks the "Join" button on an event it will toggle the `shown` state causing an inverse behavior, I added this to retoggle shown since clicking the "Join" button counts as clicking on the Event itself.

Reply:
This comment may be used to create a follow up issue in the future. Resolving to merge the PR.